addon/adapters/guid.js

```diff
@@ -0,0 +1,17 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+
+import config from 'ember-get-config';
+import DataAdapterMixin from 'ember-simple-auth/mixins/data-adapter-mixin';
+
+export default JSONAPIAdapter.extend(DataAdapterMixin, {
+    authorizer: 'authorizer:osf-token',
```

Comment:
Hmm. I wanted to avoid the OsfAdapter's extra magic, but in hindsight it shouldn't be a problem and additionally will help keep us away from divergent adapter implementations.